### PR TITLE
Reverting choice set changes for 2.2.20

### PR DESCRIPTION
**Pull request by pavankumarj**
Reverting changes done in the PR: https://github.com/BigThinkcode/AdaptiveCards/pull/66

### CODE REVIEW

File: source/community/reactnative/yarn.lock [hunk 1]
```diff
@@ -1390,7 +1390,7 @@
   resolved "https://registry.yarnpkg.com/@react-native-community/eslint-plugin/-/eslint-plugin-1.1.0.tgz"
   integrity sha512-W/J0fNYVO01tioHjvYWQ9m6RgndVtbElzYozBq1ZPrHO/iCzlqoySHl4gO/fpCl9QEFjvJfjPgtPMTMlsoq5DQ==
 
-"@react-native-picker/picker@^1.16.3":
+"@react-native-picker/picker@^1.9.11":
   version "1.16.8"
   resolved "https://registry.yarnpkg.com/@react-native-picker/picker/-/picker-1.16.8.tgz#2126ca54d4a5a3e9ea5e3f39ad1e6643f8e4b3d4"
```
Comment (Zaidos9) [inline source/community/reactnative/yarn.lock]: lets revert this as well

**Review by vivekkozhisseri — DISMISSED** [review]
LGTM

**Comment by pavankumarj** [comment]
@vivekvijayakrishnan I have pushed yarn.lock change, this is inline with the previous history, please approve the PR again.

**Review by vivekkozhisseri — APPROVED** [review]

**Review by Zaidos9 — APPROVED** [review]
LGTM!

File: source/community/reactnative/yarn.lock (hunk 1 above)
Comment (pavankumarj) [reply source/community/reactnative/yarn.lock]: Reverted all the changes in yarn.lock

**Pull request merged**